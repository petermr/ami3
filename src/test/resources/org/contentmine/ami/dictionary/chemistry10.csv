item,itemLabel,chebiId
http://www.wikidata.org/entity/Q4015903,Voacamine,10014
http://www.wikidata.org/entity/Q27176808,LSM-12060,100686
http://www.wikidata.org/entity/Q27179092,"1-cyclohexyl-3-[(2S,3R,6S)-2-(hydroxymethyl)-6-[2-(4-morpholinyl)-2-oxoethyl]-3,6-dihydro-2H-pyran-3-yl]urea",102376
http://www.wikidata.org/entity/Q27182653,"5-(3-anilinoprop-2-enylidene)-1-prop-2-enyl-1,3-diazinane-2,4,6-trione",104984
http://www.wikidata.org/entity/Q27182772,"acetic acid [2-[2-(1-oxo-3-phenylprop-2-enyl)-3-phenyl-3,4-dihydropyrazol-5-yl]phenyl] ester",105087
http://www.wikidata.org/entity/Q27182910,"N-[3-[4-(4-fluorophenyl)-1-piperazinyl]-1,4-dioxo-2-naphthalenyl]benzenesulfonamide",105206
http://www.wikidata.org/entity/Q27182987,"2,5-dichloro-N-methyl-N-[[3-(3-methylphenyl)-1,2,4-oxadiazol-5-yl]methyl]benzenesulfonamide",105274
http://www.wikidata.org/entity/Q27183045,5-[[3-(difluoromethylthio)phenyl]methyl]-2-[2-(2-pyridinylmethylidene)hydrazinyl]-4-thiazolone,105323
http://www.wikidata.org/entity/Q27183136,N-[(4-methylphenyl)methyl]-4-(1-pyrrolyl)benzamide,105409
http://www.wikidata.org/entity/Q27183259,"4-cyclopropyl-2-[[2-[(3,5-dimethyl-4-isoxazolyl)methylthio]-1-oxoethyl]amino]-3-thiophenecarboxylic acid ethyl ester",105514
