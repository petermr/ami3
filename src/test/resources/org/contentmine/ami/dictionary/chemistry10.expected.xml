<dictionary title="chemistry10">
<desc>Created from SPARQL query</desc>
<entry wikidata="Q4015903" term="Voacamine" />
<entry wikidata="Q27176808" term="LSM-12060" />
<entry wikidata="Q27179092" term="1-cyclohexyl-3-[(2S,3R,6S)-2-(hydroxymethyl)-6-[2-(4-morpholinyl)-2-oxoethyl]-3,6-dihydro-2H-pyran-3-yl]urea" />
<entry wikidata="Q27182653" term="5-(3-anilinoprop-2-enylidene)-1-prop-2-enyl-1,3-diazinane-2,4,6-trione" />
<entry wikidata="Q27182772" term="acetic acid [2-[2-(1-oxo-3-phenylprop-2-enyl)-3-phenyl-3,4-dihydropyrazol-5-yl]phenyl] ester" />
<entry wikidata="Q27182910" term="N-[3-[4-(4-fluorophenyl)-1-piperazinyl]-1,4-dioxo-2-naphthalenyl]benzenesulfonamide" />
<entry wikidata="Q27182987" term="2,5-dichloro-N-methyl-N-[[3-(3-methylphenyl)-1,2,4-oxadiazol-5-yl]methyl]benzenesulfonamide" />
<entry wikidata="Q27183045" term="5-[[3-(difluoromethylthio)phenyl]methyl]-2-[2-(2-pyridinylmethylidene)hydrazinyl]-4-thiazolone" />
<entry wikidata="Q27183136" term="N-[(4-methylphenyl)methyl]-4-(1-pyrrolyl)benzamide" />
<entry wikidata="Q27183259" term="4-cyclopropyl-2-[[2-[(3,5-dimethyl-4-isoxazolyl)methylthio]-1-oxoethyl]amino]-3-thiophenecarboxylic acid ethyl ester" />
</dictionary>
